providers/aws/resources.ts

```diff
@@ -4336,6 +4342,47 @@ export function fieldsFromElasticsearchDomainVpcOptionsParams(params: Elasticsea
   return fields;
 }
 
+export interface ElasticsearchEncryptAtRestOptionsParams {
```

Comment:
All of this is generated from `gen-providers.ts`. Did you forgot add it to commit?

Reply:
is it autogenerated?? how? i did create manually hahaha